utils/stringService.ts

```diff
@@ -68,6 +68,28 @@ export function isHexString(str: string): boolean {
   return /^0x[0123456789abcdefABCDEF]+$/.test(str);
 }
 
+// this makes sure hex string will be 64 chars
+export function formatHexString(txHash : string) {
```

Comment:
You need to test that function. 

You can use GPT https://app.skiff.com/docs/8282cddd-9cd9-420c-a636-f2d81aec00c3#adxZyhMhabLN0CVSpEqE6qRBUq65NGGPZxxHLa9qw%2F0%3D